src/Hyperwallet/Hyperwallet.php

```diff
@@ -138,8 +137,16 @@ public function updateUser(User $user) {
      *
      * @throws HyperwalletApiException
      */
-    public function listUsers($options = array()) {
-        $body = $this->client->doGet('/rest/v4/users', array(), $options);
+    public function listUsers($options=array()) {
+        if(!empty($options)) {
+            $filterOutKeys = array('status', 'clientUserId','email','programToken','verificationStatus');
+            $filteredArr = array_diff_key( $options, array_flip( $filterOutKeys ) );
```

Comment:
Would it be better to put filteredOutKeys in a constant ?

Reply:
Addressed all comments . thanks @arao6